scala-test-suite/src/test/resources/2.13.0/BlacklistedTests.txt

```diff
@@ -122,12 +172,20 @@ scala/util/matching/RegexTest.scala
 # Require strict-floats
 scala/math/BigDecimalTest.scala
 
-# Difference of getClass() on primitive values
-scala/collection/immutable/RangeTest.scala
-
-# Bugs
-scala/collection/convert/MapWrapperTest.scala
+# Fails for a BigDecimal range with augmented precision (might be an actual bug)
```

Comment:
Do we need to file an issue for this at least?

Reply:
Filed as #3706.